changelog.txt

```diff
@@ -13,6 +13,10 @@ that repo.
 Template for new versions:
 
 ## Structures
+- ``Occupation``: enum defined
```

Comment:
i'd prefer that it be `occupation_type` as our C++ style doesn't allow enums to have caps-first names

Reply:
done